packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -268,6 +263,16 @@ Future<XcodeBuildResult> buildXcodeProject({
     previewDart2: buildInfo.previewDart2,
   );
 
+  if(hasFlutterPlugins){
+    final String curGeneratedXCConfig = readGeneratedXCConfig(app.appDirectory);
```

Comment:
Don't abbreviate, just call it 'currentGeneratedXCConfig' or 'newGeneratedXCConfig'